stdlib/Test/src/Test.jl

```diff
@@ -118,18 +118,24 @@ struct Fail <: Result
     orig_expr::String
     data::Union{Nothing, String}
     value::String
+    context::Union{Nothing, String}
     source::LineNumberNode
     message_only::Bool
-    function Fail(test_type::Symbol, orig_expr, data, value, source::LineNumberNode, message_only::Bool=false)
+    function Fail(test_type::Symbol, orig_expr, data, value, context, source::LineNumberNode, message_only::Bool=false)
```

Comment:
If `context` is set to a LineNumberNode (`let source=@__SOURCE__`), it will call the wrong (the deprecated) function?

Reply:
I guess I'll remove the `=false` for now to make it explicit.